installing/installing-preparing.adoc

```diff
@@ -60,12 +59,16 @@ If you need to perform basic configuration for your installer-provisioned infras
 
-You can use the installer-provisioned infrastructure method to create appropriate machine instances on your hardware for xref:../installing/installing_openstack/installing-openstack-installer-custom.adoc#installing-openstack-installer-custom[{rh-openstack}], xref:../installing/installing_openstack/installing-openstack-installer-kuryr.adoc#installing-openstack-installer-kuryr[{rh-openstack} with Kuryr], xref:../installing/installing_rhv/installing-rhv-default.adoc#installing-rhv-default[{rh-virtualization}], xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[vSphere], and xref:../installing/installing_bare_metal_ipi/ipi-install-overview#ipi-install-overview[bare metal]. Additionally, for xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#installing-vsphere-installer-provisioned-network-customizations[vSphere], you can also customize additional network parameters during installation.
+
+You can use the installer-provisioned infrastructure method to create appropriate machine instances on your hardware for xref:../installing/installing_openstack/installing-openstack-installer-custom.adoc#installing-openstack-installer-custom[{rh-openstack}], xref:../installing/installing_openstack/installing-openstack-installer-kuryr.adoc#installing-openstack-installer-kuryr[{rh-openstack} with Kuryr], xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[vSphere], and xref:../installing/installing_bare_metal_ipi/ipi-install-overview#ipi-install-overview[bare metal]. Additionally, for xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#installing-vsphere-installer-provisioned-network-customizations[vSphere], xref:../installing/installing_vmc/installing-vmc-network-customizations.adoc#installing-vmc-network-customizations[VMC on AWS], you can also customize additional network parameters during installation.
```

Comment:
```suggestion
[...] xref:../installing/installing_vmc/installing-vmc-network-customizations.adoc#installing-vmc-network-customizations[VMC on AWS], you can also customize additional network parameters during installation.
```

This line here appears to be the specific cause for the latest Travis failure, but in general I am seeing a lot of hits in the branch for `installing/installing_vmc/` related references, but I don't see any references to `installing_vmc` in `_topic_map.yml` currently.